syntax = "proto3";

package userservice;

message EmailDaVerificare {
  string email = 1;
}

message EsitoVerifica {
  bool esiste = 1;
}

message EsitoEliminazione {
  bool successo = 1;
}

service UserService {
  rpc VerificaEmail(EmailDaVerificare) returns (EsitoVerifica);
  rpc EliminaInteresse(EmailDaVerificare) returns (EsitoEliminazione);

}